retriever/lib/defualts_fn.py

```diff
@@ -0,0 +1,62 @@
+import csv
+import imp
+import io
+import os
+import sys
+
+from retriever.lib.defaults import  ENCODING
+
+
+def open_fr(file_name, encoding=ENCODING, encode=True):
```

Comment:
Now that we have these functions here, I think we can delete them form scripts.py

Reply:
How about deleting this file and moving them to one default file?